src/Microsoft.Health.Dicom.Core/Features/Query/Model/StudyResult.cs

```diff
@@ -61,4 +61,18 @@ public DicomDataset DicomDataset
             return _dicomDataset;
         }
     }
+
+    public static readonly IReadOnlyCollection<DicomTag> AvailableTags = new HashSet<DicomTag>()
```

Comment:
I know that Josiah, Esma, Karthik, and I discussed standardizing how we refer to the different tag semantics. @josiahvinson - do you have our notes on what we'd call this scenario? lol

Reply:
I am not sure of the context. This is just exposing the indexed columns as tags